### PR TITLE
Force guava version

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: sslr-flex-toolkit/pom.xml [hunk 1]
```diff
@@ -46,35 +49,18 @@
           </archive>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>jarjar-maven-plugin</artifactId>
```
Comment (ilia-kebets-sonarsource) [inline sslr-flex-toolkit/pom.xml:51]: do we not need this plugin?

Comment (ilia-kebets-sonarsource) [reply sslr-flex-toolkit/pom.xml:51]: oh, we use shade instead of jar jar binx

**Review by ilia-kebets-sonarsource — APPROVED** [review]
LGTM, maybe indicate in the commit message that we also:
- update the sslr version
- replace the jarjar plugin with shade

File: sslr-flex-toolkit/pom.xml (hunk 1 above)
Comment (saberduck) [reply sslr-flex-toolkit/pom.xml:51]: yes

**Pull request merged**